vault/dendron.ref.config.publishing.md

```diff
@@ -0,0 +1,138 @@
+---
+id: q960Z622o0RLO32w0B8SP
+title: Publishing
+desc: ''
+updated: 1645183292834
+created: 1644903086084
+config:
+  global:
+    enableChildLinks: false
+---
+
+***
```

Comment:
do we need a line break here? why isn't the header alone enough? also, should the section be an h3 title vs a h2 title?

Reply:
> do we need a line break here? why isn't the header alone enough?

To make a more visually distinct delineation between different entries. I can remove them if you don't think that's a good reason.

> also, should the section be an h3 title vs a h2 title?

The title of the note will be h1, wouldn't it be natural that these are h2?
Also smaller headers make it harder to visually delineate

Reply:
i generally go with h3 headers for definitions but don't feel strongly about this. for the line break, i would prefer not to have them since it adds visual noise and we would also need to add it in other configurations to be consistent

Reply:
Got it. I'll push the update in a bit.